### PR TITLE
style: optimize select dropdown show in rtl

### PR DESCRIPTION
**Pull request by xrkffgg**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
none

### 💡 Background and solution
image

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Optimize Select dropdown show in RTL        |
| 🇨🇳 Chinese | 优化 Select 下拉选 RTL 模式下文字展示          |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/select/style/rtl.less [hunk 1]
```diff
@@ -28,7 +28,8 @@
   // ========================== Popup ==========================
   &-dropdown {
     &-rtl {
-      direction: rtl;
+      direction: ltr;
```
Comment (yoyo837) [inline components/select/style/rtl.less:31]: 如果刚好下拉内容是RTL文字呢？

Comment (xrkffgg) [reply components/select/style/rtl.less:31]: 我觉得没啥问题，类似于这个一样 https://github.com/ant-design/ant-design/issues/22555

Comment (xrkffgg) [reply components/select/style/rtl.less:31]: 有些地方的展示，我感觉，数字类，x.x 类，direction: rtl; 
完全意义就不同了，这或许和 这个属性有关。

Comment (hengkx) [reply components/select/style/rtl.less:31]: ref #23207

**Pull request merged**